### PR TITLE
adder collator updates

### PR DESCRIPTION
**Pull request by pepyakin**
- Fix a typo in README
- Log some collation progress to demonstrate that it's alive.

### CODE REVIEW

**Review by ordian — APPROVED** [review]

**Comment by ordian** [comment]
bot merge

File: parachain/test-parachains/adder/collator/src/lib.rs [hunk 1]
```diff
@@ -100,12 +100,18 @@ impl Collator {
...
+		Box::new(move |relay_parent, validation_data| {
 			let parent = HeadData::decode(&mut &validation_data.persisted.parent_head.0[..])
 				.expect("Decodes parent head");
 
 			let (block_data, head_data) = state.lock().unwrap().advance(parent);
 
+			log::info!(
+				"created a new collation on relay-parent {} ({})",
+				validation_data.persisted.block_number,
+				relay_parent,
```
Comment (bkchr) [inline parachain/test-parachains/adder/collator/src/lib.rs:112]:
```suggestion
				"created a new collation on relay-parent({}): {:?}",
				relay_parent,
				block_data,
```

**Review by bkchr — APPROVED** [review]

**Comment by pepyakin** [comment]
bot merge cancel

**Comment by pepyakin** [comment]
bot merge

**Pull request merged**